cmd/destroy/remote.go

```diff
@@ -242,7 +243,25 @@ func (rd *remoteDestroyCommand) createDockerignore(cwd, tmpDir string) error {
 			return err
 		}
 	}
-	return afero.WriteFile(rd.fs, fmt.Sprintf("%s/%s", tmpDir, ".dockerignore"), dockerignoreContent, 0600)
+
+	// write the content into the .dockerignore used for building the remote image
+	filename := fmt.Sprintf("%s/%s", tmpDir, ".dockerignore")
```

Comment:
Maybe not to address on this PR but we need to have a common place for deploy and destroy

Reply:
I believe the whole remote thing can be in a common place as we do have a lot of duplicates

Reply:
i've added changes to unify at least the dockerignore logic